maps/Collection.json

```diff
@@ -36,6 +36,10 @@
       "description": "Identifier for the resource within SDR but outside of TACO. DRUID or UUID depending on resource type. Constant across resource versions. What clients will use calling TACO. Same as `identification.identifier`",
       "type": "string"
     },
+    "sourceId": {
```

Comment:
per identification work on these + related questions a few weeks ago, you shouldn't be adding + using sourceId at the top level, but using the now existing 'dedupeIdentifiers'.

Reply:
See https://github.com/sul-dlss-labs/taco/wiki/TACO-&-SDR3-Identifier-schema#other-identifiers